client/me/style.scss

```diff
@@ -0,0 +1,101 @@
+@media only screen and (min-width: 601px) {
+	html {
+		overflow-y: hidden;
```

Comment:
This breaks the scrolling.

I started experiencing random freezes of the screen when interacting with the /stats page and the reader with scrolling not working. I reached out to @tyxla with the report of this strange and hard to reproduce behaviour and the next time it froze, I noticed that this style is applied. 

@tyxla managed to figure out the steps to reliably reproduce based on looking at this code here
- Go to /stats (or /plans or any other similar page that has scroll)
- Click on your avatar in the masterbar
- Click the arrow that points back

It otherwise happens to me randomly when I use the site anywhere. 

This is a breaking change and we should consider reverting.

Reply:
I confirm that I can also reproduce consistently by going through the same steps.